admin/metabox/class-metabox.php

```diff
@@ -369,6 +376,23 @@ private function get_content_sections() {
 		return $content_sections;
 	}
 
+	/**
+	 * Returns the metabox content for React to hook into.
+	 *
+	 * @return WPSEO_Metabox_Section
+	 */
+	private function get_content_meta_section_react() {
```

Comment:
this method returns content that is specific to the "Content optimization" section, shouldn't be named accordingly? the current name and doc make me think it's a generic section.

Reply:
The "content" in "content meta section" refers to the "Content optimization".